### ISSUE TITLE
Update feature table to show/navigate related records

### ISSUE THREAD

**Issue by azizaparveen** [Esri/solutions-components#762]
### Describe requirements

Last release Map Viewer added an update that improved viewing related records in the table.
API reference : https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-FeatureTable.html#related-records
![image](https://github.com/user-attachments/assets/2ef2d878-052f-492b-a8c0-de06bd5c49fa)
![image](https://github.com/user-attachments/assets/4e73808f-5f78-454d-b0b2-2a7aa925701a)
Sample web map with related data: https://arcgis4localgov2.maps.arcgis.com/apps/mapviewer/index.html?webmap=80c1039322cc48baa0d08b522694a96c

**Comment by sumitzarkar** [comment]
@azizaparveen 

- Will this be a configurable option so that users can opt it for backward compatibility or we want to show related record colunm by default

**Comment by azizaparveen** [comment]
@sumitzarkar I don't think we need configuration for this and would like to show the column by default

**Comment by chris-fox** [comment]
We might need to think about the UX here a bit:

- We have a toolbar that has many buttons that interact with the active table, when viewing related records should the buttons work against the related table (Select All, Clear Selection, Export, Delete, Filter, etc.)
- When you select a record in the related table are you viewing its pop-up in the details. Is it editable?

**Comment by azizaparveen** [comment]
@sumitzarkar I discussed this user experience issues Chris mentioned here, and we decided to push this to backlog.
